packages/strapi-design-system/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { After } from '@strapi/icons';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { useId } from '../helpers/useId';
+
+const CrumbWrapper = styled.li`
+  display: inline-flex;
+  align-items: center;
+  svg {
+    height: 10px;
+    width: 10px;
+  }
+  svg path {
+    fill: ${({ theme }) => theme.colors.neutral300};
+  }
+`;
+
+export const Crumb = React.forwardRef(({ children, isLastChild }, ref) => {
+  return (
+    <CrumbWrapper ref={ref}>
+      <Text highlighted color="neutral800">
+        {children}
+      </Text>
+      {!isLastChild && (
+        <Box paddingLeft={3} paddingRight={3}>
+          <After />
+        </Box>
+      )}
+    </CrumbWrapper>
+  );
+});
+Crumb.displayName = 'Crumb';
+Crumb.defaultProps = {
+  isLastChild: false,
+};
+Crumb.propTypes = {
+  children: PropTypes.string.isRequired,
+  isLastChild: PropTypes.bool,
+};
+
+export const Breadcrumbs = React.forwardRef(({ children, ...props }, ref) => {
+  const breadCrumbsId = useId('breadcrumbs');
+  const validChildren = React.Children.toArray(children).filter((child) => React.isValidElement(child));
```

Comment:
What is the aim of the "isValid" call?

Reply:
Just to add validation of the children in order to have valid elements. We can remove it if you think that we don't need it.

Reply:
I don't think you need it here, such logic could be interesting if you were cloning the children to add extra props.